AmazonChimeSDKDemo/AmazonChimeSDKDemo/MeetingViewController.swift

```diff
@@ -392,7 +393,17 @@ class MeetingViewController: UIViewController {
                                                 meetingModel.setVoiceFocusEnabled(enabled: !isVoiceFocusEnabled)
                                              })
         optionMenu.addAction(voiceFocusAction)
-
+        
+        let isLiveTranscriptionEnabled = meetingModel.captionsModel.isLiveTranscriptionEnabled
+        let nextLiveTranscriptionStatus = nextOnOrOff(current: isLiveTranscriptionEnabled)
+        let liveTranscriptionAction = UIAlertAction(title: "Turn \(nextLiveTranscriptionStatus) Live Transcription",
+                                             style: .default,
+                                             handler: { _ in
+                                                meetingModel.setLiveTranscriptionEnabled(enabled: !isLiveTranscriptionEnabled)
+                                             })
+        isLiveTranscriptionOn = !isLiveTranscriptionEnabled
```

Comment:
Is this `isLiveTranscriptionOn` being used anywhere?

Reply:
Np, removed it.